src/schema.js

```diff
@@ -174,6 +173,24 @@ class Schema {
     });
   }
 
+  /**
+   * Promise returned by invoking generated API methods.
+   * @interface GeneratedPromise
+   * @type Promise<Object>
+   * @property {Number} requestId The JSON-RPC request id.
+   */
+
+  /**
+    * API method for a specific schema method. Note that this is not
```

Comment:
not really the best way to document this - the entire generated api is an object of type `object<string,generatedMethod>` with some additional custom properties that override the default property type of the object. The jsdoc for that might look something like this:

```js
/**
 * @interface api
 * @type {object<string,generatedMethod>}
 * @property {string} id
 * ...
 */
```

Unfortunately this is not something jsdoc nor scriptappy supports out of the box

Reply:
Gotcha, tried that as well. Since that doesn't work, the way it's done in this PR _is_ the best _workable_ way, or?

Reply:
I think the best workable solution right now is to not include it in the spec at all, rather than including a wrong spec.

Reply:
Agreed, if that's acceptable. @wennmo @hrigner ?